MontyHallTests/ConsoleGameTests.cs

```diff
@@ -0,0 +1,162 @@
+using MontyHallv2.DoorCreation;
+using MontyHallv2.GameModes;
+using MontyHallv2.GameShowStaff;
+using MontyHallv2.Interfaces;
+using MontyHallv2.Messages;
+using MontyHallv2.Random;
+using Moq;
+
+namespace TestProject1.moq;
+
+public class ConsoleGameTests
+{
+    private readonly List<Door> _doors;
+    private readonly ConsoleGame _consoleGame = new();
+    private const int CarDoor = 0; 
+
+    public ConsoleGameTests()
+    {
+        //Assert
+        _doors = Enumerable.Range(1, 3)
+            .Select(_ => new Door())
+            .ToList();
+        
+        _doors[CarDoor].InjectCarToDoor();
```

Comment:
```suggestion
        _doors[PrizeDoor].InjectCarToDoor();
```
Does this read nicely? Will comment on this more later :)

Reply:
understood that cardoor can literally mean a physical car door, prizeDoor makes much more sense. thank you